components/PickerBase/PickerBase.vue

```diff
@@ -23,14 +23,17 @@
               >*</sup
             >
           </div>
-          <BButton
-            v-if="showAllButton"
-            id="picker-all-button"
-            data-cy="picker-all-button"
-            size="sm"
-            variant="primary"
-            v-on:click="pickAll()"
-            >All</BButton
+         <BButton
+           v-if="showAllButton"
+           id="picker-all-button"
+           data-cy="picker-all-button"
+           size="sm"
+           :variant="checked.length === options.length ? 'danger' : 'primary'"
+           v-on:click="pickAll"
+          >
+           <span v-if="checked.length === options.length">🚫 All</span>
+           <span v-else>✅ All</span>
+         </BButton>
           >
```

Comment:
There is an extra `>` here that is showing up in the UI and it should be removed.